### PR TITLE
(DO NOT MERGE)(MODULES-6989) Add multiple Linux CSR extensions 

### PR DESCRIPTION
**Pull request by Iristyle**
Builds on the task rename from #18 which should be merged first

 - Modify the metadata for custom_attribute and extension_request to
   accept and Array[Pattern] and not simply a Pattern to be consistent
   with the Windows task

 - This relies on the `jq` command line utility being installed as a
   pre-requisite as a cross-platform friendly awk / sed invocation is
   difficult to implement.

### CODE REVIEW

File: tasks/linux.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ certname="$PT_certname"
 alt_names="$PT_dns_alt_names"
 custom_attribute="$PT_custom_attribute"
 extension_request="$PT_extension_request"
+jq_missing="jq is required to parse CSR custom attributes or extension requests"
```
Comment (Iristyle) [inline tasks/linux.sh]: This seems like an ideal spot to bring in an additional script like one of:

* [TickTick](https://github.com/kristopolous/TickTick) - licensed as MIT *or* Apache 2
* [JSON.sh](https://github.com/dominictarr/JSON.sh) - licensed as MIT *or* Apache 2
* [JSONPath.sh](https://github.com/mclarkson/JSONPath.sh) - licensed as MIT *or* Apache 2
* [JSON.awk](https://github.com/step-/JSON.awk) - licensed as MIT *or* Apache 2

Not sure if you've given this any thought yet @MikaelSmith / @adreyer ?

Comment (MikaelSmith) [reply tasks/linux.sh]: Those would be interesting. We need a way to include multiple files in a task, or a packaging step to merge everything into one file.

If we had multiple files in a task, jq is portable enough we could probably include the binary.

**Comment by pmcmaw** [comment]
Hey @Iristyle
Just wondering if there is any movement on this PR?

**Comment by pmcmaw** [comment]
I am closing this PR due to lack of movement, if you disagree with my action please feel free to reopen.

Many Thanks
The Modules Team

**Pull request closed without merging**